### PR TITLE
[relay]: Cleansup authors

### PR DESCRIPTION
**Pull request by maraisr**
This PR aims to clear out tagging people in PR's that haven't touched the types since 2019. We need to foster a healthy small, but solid group of maintainers for this package—it'll helper iterate quick as we need to iterate almost as fast as the Facebook folk.

cc @alloy Maybe I don't understand DT, but happy to leave this here from a "people who's done some work on it", but afaik the bot tags all those people, and nobody responds.

### CODE REVIEW

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## react-relay ([<kbd>unpkg</kbd>](https://unpkg.com/browse/react-relay@latest/))
was missing the following properties:
1. ConnectionHandler
1. MutationTypes
1. RangeOperations
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 32011cb3be377156f4b1f1e8f360a19b6d5de009
</p>

**Review by Stephen2 — APPROVED** [review]

**Comment by Stephen2** [comment]
Sorry I haven't had time to contribute more in last couple months 👍

**Comment by maraisr** [comment]
That is more than okay @Stephen2 we should keep you, if you're still interested in helping! I'm just super keen on having people part of it that truly want to be. We all have seasons, time is precious, and OSS is hard—i'll probably also going through a season of not doing anything.

**Comment by Stephen2** [comment]
Thanks, keep me deleted for sure.  Hopefully I can come back.

**Comment by elibarzilay** [comment]
Seems like a good idea -- if any of the dropped owners sees this and wants to stay they can readd themselves.  So just because I'm too lazy to do the set subtraction (which will hopefully be added soon as something that the bot will do), here's a quick JFYI ping for the owners...

@Graphcool @voxmatt @alloy @npirotte @ckknight @kastermester @mattkrick @jdk243 @renanmav @janicduplessis @ChristianIvicevic @edvinerikson @Stephen2 @morrys

**Comment by alloy** [comment]
I would personally have preferred to keep a larger list of people that have stepped in every now and then, just as long as they understood what it meant at the time they added their name to the list (which all these people did because I always asked). Now we made the chance that they will step in at times that others of us don’t have time close to none.

Perhaps you thought the list is one that requires sign-off by more people on the list, but that’s not the case. What it is, to me, is a list of people that I will trust to have reviewed a PR well enough for me to spend little to no time reviewing before merging.

I’m a little on the fence of what to do now. I don’t think all people need to be added back, but people like the following would have my preference to still be listed: @voxmatt @kastermester @renanmav @janicduplessis @morrys

**Comment by morrys** [comment]
Hi @maraisr,
I understand the need to update the maintainers of the library but I would like to make a couple of considerations as some of the people who have been eliminated updated the types in 2020 (myself included).
Also, I would have given people more time to respond to PR as there are different time zones.

As for me, it was convenient for me to be on the list because they maintain more than one library that relies on relay and I feel comfortable staying up to date on type issues. Of course, time permitting, I could also do the revisions but my best contribution is to solve typing problems that I encounter during development / maintenance of libraries.

**Comment by maraisr** [comment]
Everyone. I made a mistake contributing here. Didn't realise it was a contentious ecosystem. @morrys I apologies for this mistake on my part. However, don't believe simply wanting to "subscribe to changes" is warranted being on the author list. 

As what was said, just yourself back in. I'm removing myself nevertheless.

**Comment by alloy** [comment]
> Also, I would have given people more time to respond to PR as there are different time zones.

I just want to point out here that this was done by a third party.

There were only good intentions here on everybody’s part and it's not like we cannot easily [partially] revert any change.

@morrys So it sounds like you want on that list, in which case please send a PR to add yourself back 👍

**Comment by morrys** [comment]
I want to clarify that I am not angry but I just wanted to give my opinion.
If you want to talk about it, you can find me on slack as this is not the right place 👍

**Comment by alloy** [comment]
> Everyone. I made a mistake contributing here. Didn't realise it was a contentious ecosystem. @morrys I apologies for this mistake on my part. However, don't believe simply wanting to "subscribe to changes" is warranted being on the author list.
> 
> As what was said, just yourself back in. I'm removing myself nevertheless.
> 
> I tried to do a positive thing here, and foster a healthy maintainer base for the Relay packages. Because quite frankly, it was dead... and all I'm receiving is negativity.
> 
> I did know raising this PR would cause some raised eyebrows, so not entirely surprised there are upset people. But quite frankly, I've raised that many fixes recently in this space. And have had a single person help me get that merged....
> 
> Sorry DT maintainers I'm probably breaking some of the code of conduct rules—so will stay silent now.

I find this a most unfortunate outcome. Your work is most appreciated by me and presumably all TS users of Relay.

As I mentioned in our private chat, we should be able to disagree about things and work to a mutually agreed upon path forward, which often involves concessions from most parties. Being able to raise things we want to discuss is important for a healthy collaborative environment with the assumption, to a certain extent (!), that those points are not interpreted as hostile/personal attacks/the different nuanced way you are perceiving this.

I do not believe I or others have shown any [harsh] negativity or would even label it contentious (although I guess that might brings us into semantic discussion area, which is probably best to stay away from in a medium like this).

FWIW As I said before, _I_ do not expect any apologies or for you to even feel like needing to say so. There was simply **no** wrongdoing here. I get your “move fast” motivation, love the energy, and would want to keep that going, but likewise it would then also be good for you to understand other perspectives and their context. I have offered to jump on a call and that offer still stands.

**Comment by sibelius** [comment]
I think the more reviewers the better

**Comment by elibarzilay** [comment]
Apologies for merging it after only 2 days, I should have probably waited more time.  (We have a week cycle, so there's some motivation to get things done and minimize leftovers for the next week.)

In any case, if people want to revert the whole thing, I can go over the replies and revert the change except for people who said that they want to be removed.

In addition to this, I'm working on some changes to the bot that will make it clearer when people add themselves.  (DefinitelyTyped/dt-mergebot#202).

**Comment by alloy** [comment]
@elibarzilay As said before, but just for the record, there were no wrongdoings by anybody here. Thanks for working on the bot to address these things more broadly 🙏

**Comment by sorenhoyer** [comment]
I never added myself to the list, despite a few contributions in the past, but I do think it is very bad practice to delete people who are listed without first reaching out to them and getting their acceptance. If they had not contributed / helped maintaining (even if just a little bit) in the past, perhaps this package would not be where it is today.

Btw the EntryPoint types are out of date and needs to be updated.

**Pull request merged**
